lib/tableau/extensions/post_extension/posts/post.ex

```diff
@@ -10,6 +10,12 @@ defmodule Tableau.PostExtension.Posts.Post do
     |> Map.put(:body, body)
     |> Map.put(:file, filename)
     |> Map.put(:layout, Module.concat([attrs.layout || post_config.layout]))
+    |> Map.put_new_lazy(:title, fn ->
```

Comment:
Can you add this to the PageExtension as well? (I just made it today, lol)

Reply:
Should we extract these to a common module, related to YAML frontmatter posting? They seem useful enough as a "generic" thing

Reply:
Potentially, but I think it's fine to stay duplicated for now